### PR TITLE
[TR] Update Mascots

### PR DESCRIPTION
**Pull request by zentoryny**
deneme

### CODE REVIEW

File: wiki/Mascots/tr.md [hunk 1]
```diff
@@ -1,231 +1,25 @@
 ---
-outdated: true
+outdated: false
 ---
```
Comment (TPGPL) [inline wiki/Mascots/tr.md:4]: Remove this completely

File: wiki/Mascots/tr.md [hunk 2]
```diff
@@ -1,231 +1,25 @@
...
 
 # Mascots (Türkçe)
```
Comment (TPGPL) [inline wiki/Mascots/tr.md:5]: needs a proper translation

```
# <translation of Mascots word>
```

File: wiki/Mascots/tr.md [hunk 3]
```diff
@@ -1,231 +1,100 @@
----
-outdated: true
----
```
Comment (TPGPL) [inline wiki/Mascots/tr.md:1]: remove this blank line

File: wiki/Mascots/tr.md [hunk 4]
```diff
@@ -1,231 +1,99 @@
...
+*Haber gönderileri için, bknz: [Yuzu ile tanışın!](https://osu.ppy.sh/home/news/2014-06-21-meet-yuzu) ve [Yuzu'nun Yeni Görünümüyle Tanışın](https://osu.ppy.sh/home/news/2019-01-09-introducing-yuzu)*
 
 ![Yuzu](img/Yuzu.png "Yuzu")
 
-#### Gerçek adı
-Yuzu
-
-#### Takma Adlar
-Yuzu
+Yuzu, 2014-06-22 tarihinde katılan osu catch maskotudur. 2000-04-10 tarihinde doğdu, 172 santimetre boyunda, ve 65 kilo ağırlığında. şu anki tasarımı [Thievley](https://osu.ppy.sh/users/4717672) tarafından tasarlanmıştır. Oysa il tasarımı ve yakalıyıcı sprite'ı [ztrot](https://osu.ppy.sh/users/6347) tarafından yapılmıştır; Daru birleşik patlama sanatını yarattı.
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:21]: Biraz rötuşladım
```suggestion
Yuzu, 2014-06-22 tarihinde katılan osu!catch maskotudur. 2000-04-10 tarihinde doğdu, 172 santimetre boyunda, ve 65 kilo ağırlığında. şu anki tasarımı [Thievley](https://osu.ppy.sh/users/4717672) tarafından tasarlanmıştır. Oysa ilk tasarımı ve yakalayıcı sprite'ı [ztrot](https://osu.ppy.sh/users/6347) tarafından yapılmıştır; Daru kombo patlaması resmini oluşturdu.
```

File: wiki/Mascots/tr.md [hunk 5]
```diff
@@ -1,231 +1,99 @@
...
+Haber gönderileri için, bknz: [Maria ile tanışın - osu!mania’nın yeni maskotu!](https://osu.ppy.sh/home/news/2016-04-20-meet-maria-osumanias-new-mascot)*
 
 ![Maria](img/Maria.png "Maria")
 
-#### Gerçek Adı
-Maria
-
-#### Takma adları
-Maria
+Maria, 2016-03-04 tarihinde katılan osu! Mania maskotudur. O Daru tarafından tasarlanmıştır.
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:29]:
```suggestion
Maria, 2016-03-04 tarihinde katılan osu! Mania maskotudur. Kendisi Daru tarafından tasarlanmıştır.
```

File: wiki/Mascots/tr.md [hunk 6]
```diff
@@ -1,231 +1,99 @@
...
-?
+Haber gönderileri için, bknz: [Yeni osu!taiko maskotu burada!](https://osu.ppy.sh/home/news/2017-05-25-the-new-osutaiko-mascot-is-here)*
 
-#### Belirmeler
-Mania
+![Mocha](img/Mocha.png "Mocha")
 
-#### Yaratıcı
-- Daru (Çizim)
+Mocha osu!taiko maskotu. O başta [sixth fanart contest](https://osu.ppy.sh/community/contests/2) esnasında [Crowie](https://osu.ppy.sh/users/6894067) tarafından tasarlandı, anketlerde 21. sırada yer aldı.
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:37]:
```suggestion
Mocha, osu!taiko maskotu. Aslen ilk kez [altıncı fanart yarışması](https://osu.ppy.sh/community/contests/2) esnasında [Crowie](https://osu.ppy.sh/users/6894067) tarafından tasarlanmıştır, oylamada ise 21. sırada yer almıştır.
```

File: wiki/Mascots/tr.md [hunk 7]
```diff
@@ -1,231 +1,99 @@
...
-
-#### Boy
-?
-
-#### Belirmeler
-- Catch the Beat (as chibi-Ryuuta) [Yuzu ile değiştirildi]
-- osu!webpage (chibi-Ryuuta as Mascot) [Emekli oldu]
-- [Skin](https://osu.ppy.sh/community/forums/topics/190357/)
-- Yandere Simulator (Easter egg bir NPC olarak)
+> O tezahürat ekibinin sıcakkanlı lideridir. İyi bir ruhu vardır ve çevresinde sıkıntı içinde olanlar için ayağa kalkıp onlar için savaşır!
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:45]:
```suggestion
> Kendisi tezahürat ekibinin sıcakkanlı lideridir. İyi bir ruhu vardır ve çevresinde sıkıntı içinde olanlar için ayağa kalkıp onlar için savaşır!
```

File: wiki/Mascots/tr.md [hunk 8]
```diff
@@ -1,231 +1,99 @@
...
 
-#### Yaratıcı
-iNiS Corporation
+一本木龍太 (Ryūta Ippongi) 2008'de katılmış olan osu!catch chibi-fruit catcher'dır, ancak yerini 2014 yılında [Yuzu](#-yuzu) aldı. O [iNiS Corporation](https://en.wikipedia.org/wiki/INiS) tarafından oluşturulmuştur ve bir zamanlar eski web sitesinin bir parçasıydı. Ayrıca [Yandere Simulator](https://yanderesimulator.com)' de NPC'dir.
 
-#### Cinsiyet
-Erkek
+Ryuuta [LuigiHann](https://osu.ppy.sh/users/1079)'nın skin'inde de yer aldı, [Elite Beat osu! HD (1.0 Complete!)](https://osu.ppy.sh/community/forums/topics/190357/).
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:49]:
```suggestion
Ryuuta, [LuigiHann](https://osu.ppy.sh/users/1079)'nın temasında da yer aldı, [Elite Beat osu! HD (1.0 Complete!)](https://osu.ppy.sh/community/forums/topics/190357/).
```

File: wiki/Mascots/tr.md [hunk 9]
```diff
@@ -1,231 +1,99 @@
...
 
-#### Biyoveri
-Moral takımının sıcak kanlı lideri. İyi bir kalbi vardır ve etrafındakiler zor duruma düştüğünde onların yanında savaşır!
+> Hip-hop'tan bale'ye kadar birçok dans stilinde uzman, J herhangi bir canlıyı büyüleyebilir.
 
-### Agent J
+Ajan J, BA-2 (Beat Agent-2) veya J olarak da bilinir, 2008'de osu! maskotlarından biriydi ama 2014'te emekli olmuştur. İNiS Corporation tarafından oluşturuldu ve bir zamanlar eski web sitesinin bir parçasıydı.
 
-![Agent J](img/Agent_J.png "Agent J")
+Ajan J [LuigiHann](https://osu.ppy.sh/users/1079)'nın skin'inde de yer aldı [Elite Beat osu! HD (1.0 Complete!)](https://osu.ppy.sh/community/forums/topics/190357/).
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:59]:
```suggestion
Ajan J, [LuigiHann](https://osu.ppy.sh/users/1079)'nın temasında da yer aldı [Elite Beat osu! HD (1.0 Complete!)](https://osu.ppy.sh/community/forums/topics/190357/).
```

File: wiki/Mascots/tr.md [hunk 10]
```diff
@@ -1,231 +1,99 @@
...
 
-#### Biyoveri
-Hip-hop'tan baleye, pek çok dans stilinde uzman olan J, yaşayan herkesi etkisi altına alabilir.
+### Aiko
 
-### Don
+![Aiko](img/Aiko.png "Aiko")
 
-![Don](img/Don.png "Don")
+[JMC](https://osu.ppy.sh/users/774010) tarafından tasarlanmıştır. Aiko, osu! Taiko maskot tasarım yarışmasına katılanlardan biriydi. O Osu!Taiko için tutkulu ve enerjik bir kızdır, o en iyisi olmasa bile! Bir çift şık "Tabi" spor ayakkabısı ve pippidon aksesuarları beraberliğinde, yıpranmış maskot bu cesur kızda yaşıyor. Sadece 154 santimetrede oldukça kısadır, ve 1999-04-06 tarihinde doğmuştur.
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:75]:
```suggestion
[JMC](https://osu.ppy.sh/users/774010) tarafından tasarlanmıştır. Aiko, osu!taiko maskot tasarım yarışmasına katılanlardan biriydi. O osu!taiko için tutkulu ve enerjik bir kızdır, o en iyisi olmasa bile! Bir çift şık "Tabi" spor ayakkabısı ve pippidon aksesuarları beraberliğinde, yıpranmış maskot bu cesur kızda yaşıyor. Sadece 154 santimetrede oldukça kısadır, ve 1999-04-06 tarihinde doğmuştur.
```

File: wiki/Mascots/tr.md [hunk 11]
```diff
@@ -1,231 +1,99 @@
...
-和田どん (Wada Don)
+### Alisa
 
-#### Takma Adları
-Don, Don-chan
+![Alisa](img/Alisa.png "Alisa")
 
-#### Doğum tarihi
-?
+[\[ Glitch \]](https://osu.ppy.sh/users/3781400) tarafından tasarlanmıştır. Alisa, osu!Taiko maskot tasarım yarışmasına katılanlardan biriydi. Çocukluğundan beri osu taiko oynuyor, başkaları için şarkı çalmaya düşkün, çok müzikal olarak yönlendirilmiş, ve uyumadığında ya da yemek yemediğinde, osu! taiko oynuyor, veya tekmeler için retro video oyunları oynuyor!
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:81]:
```suggestion
[\[ Glitch \]](https://osu.ppy.sh/users/3781400) tarafından tasarlanmıştır. Alisa, osu!taiko maskot tasarım yarışmasına katılanlardan biriydi. Çocukluğundan beri osu!taiko oynuyor, başkaları için şarkı çalmaya düşkün, çok müzikal olarak yönlendirilmiş, ve uyumadığında ya da yemek yemediğinde, osu!taiko oynuyor, veya tekmeler için retro video oyunları oynuyor!
```

File: wiki/Mascots/tr.md [hunk 12]
```diff
@@ -1,231 +1,99 @@
...
 
-#### Belirmeler
-- Taiko (Skin)
-{[Option'un tetiği](/wiki/Options#Skin) için resmi skin}
+![Chirou](img/Chirou.png "Chirou")
 
-#### Yaratıcı
-- Yukiko Yokoo (横尾有希子) [Çizim]
-- Narahashi Miki (楢橋 美紀) [Seslendirme]
+[pyun](https://osu.ppy.sh/users/981534) tarafından tasarlanmıştır. Chirou osu!Taiko maskot tasarım yarışmasına katılanlardan biriydi, o Çok katı ve talepkardır, mükemmeliyetçi, ve hata yapmayı sevmez - özellikle osu!taiko'daki atımlarında. Ama güçlü bakış açısını aşarak, onun yumuşak noktasına gidersen, yumuşacık ve sevimli olabilir. O 14 yaşında. 25 Ekim'de AB kan grubu ile doğdu. Ayrıca 4 feet 11 inç ve 100 kilo ağırlığında. Hobileri arasında davul çalma, pelerininde sıcak durmak, ve taş veya kayaların toplanması. Chirou, çeşitli kullanıcı fanart başvurularında bazı görünümler aldı.
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:87]:
```suggestion
[pyun](https://osu.ppy.sh/users/981534) tarafından tasarlanmıştır. Chirou osu!taiko maskot tasarım yarışmasına katılanlardan biriydi, kendisi çok katı ve talepkardır, mükemmeliyetçi, ve hata yapmayı sevmez - özellikle osu!taiko'daki vuruşlarında. Ama sert görünüşünün ardında, onun yumuşak noktasına gidersen, yumuşacık ve sevimli olabilir. O 14 yaşında. 25 Ekim'de AB kan grubu ile doğdu. Ayrıca 4 feet 11 inç ve 100 kilo ağırlığında. Hobileri arasında davul çalma, pelerininde sıcak durmak, ve taş veya çakıl toplamak bulunur. Chirou, çeşitli kullanıcı fanart başvurularında bazı görünümler aldı.
```

File: wiki/Mascots/tr.md [hunk 13]
```diff
@@ -1,231 +1,99 @@
...
-Erkek
+### Taikonator
 
-#### Katıldığı Tarih
-Mayıs 2008
+![Taikonator](img/Taikonator.png "Taikonator")
 
-#### Kilo
-100+ kg
+Taikonator, Taikonator 3000 olarak da bilinir, osu! taiko maskot tasarım yarışmasına katılanlardan biriydi, Bilinmeyen nedenlerden dolayı şaka ve popülerlik olarak biraz çekiş kazandı. Kökeni bir sır olarak kalıyor; buna rağmen, diğer osu! taiko maskot tasarım başvurularından çok daha eşsiz ve çeşitli kullanıcı fanart gönderimlerinde birçok görünüşe sahiptir.
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:93]:
```suggestion
Taikonator, Taikonator 3000 olarak da bilinir, osu!taiko maskot tasarım yarışmasına katılanlardan biriydi. Bir takım nedenlerden dolayı bazı gruplar içerisinde içsel şaka ile beraber popülerlik kazandı. Kökeni bir sır olarak kalıyor; buna rağmen, diğer osu!taiko maskot tasarım başvurularından çok daha eşsiz ve çeşitli kullanıcı fanart gönderilerinde birçok görünüşe sahiptir.
```

File: wiki/Mascots/tr.md [hunk 14]
```diff
@@ -1,231 +1,99 @@
...
 
-#### Biyoveri
-Don Taiko no Tatsujin serisinin ana karakteri. Kendisi beyaz kenarlı, kırmızı suratlı ve açık mavi vücütlu 4 bacaklı bir taiko davuludur. Don'un rüyası Taiko'nun güzelliğini dünya ile paylaşmaktır. Wada House'a taşındığından beri 3 yıl geçti, ve şehirde oldukça popüler oldu. Korkunç bir iştahı vardır ve sonu kötü bitebilecek pahalı alışverişler yapar. Cümlelerini genellikle Japoncada "Ba-dum" anlamına gelen "Ta-don" ile bitirir.
+### Tama
 
-## Resim Galerisi
+![Tama](img/Tama.png "Tama")
 
-[Galeriye gitmek için buraya tıklayın tıklayın.](/wiki/Mascots/Gallery)
+[crystalsuicine](https://osu.ppy.sh/users/9974) tarafından tasarlanmıştır. Tama, osu! Taiko maskot tasarım yarışmasına sadece 15 yaşında genç katılımcılardan biriydi (ya da öyle mi?), Tama taiko drumming için büyük bir tutkusu vardır. Ve thunderstorms. Ve özellikle, festivaller de herhangi bir yerde bulabileceği herhangi bir takoyaki kapar. Her zaman meydan okumaya hazır, Tama, genç tutumunun arkasında özellikle gizemli bir geçmişi saklıyor.
```
Comment (zeusminus) [inline wiki/Mascots/tr.md:99]:
```suggestion
[crystalsuicine](https://osu.ppy.sh/users/9974) tarafından tasarlanmıştır. Tama, osu!taiko maskot tasarım yarışmasına sadece 15 yaşında genç katılımcılardan biriydi (ya da öyle mi?), Tama'nın taiko drumming'e karşı büyük bir tutkusu vardır. Ve gök gürültülü fırtınalara. Ve özellikle de herhangi bir yerde takoyaki bulabileceği festivallere. Her zaman meydan okumaya hazır, Tama, genç görünüşünün arkasında gizemli bir geçmişi saklıyor.
```

**Pull request merged**